### DISCUSSION TITLE
Suspensión de cuenta educativa

### DISCUSSION THREAD

**Discussion by m-felipe1** [community/community#120004] [GitHub Education]
### Hi everyone,

Tengo una cuenta creada con un correo institucional la cual solo uso para el paquete de estudiantes y ciertos beneficios que ofrece pero de un momento a otro resulta que dicha cuenta ha sido suspendida y no se el motivo y no se como recuperarla. Al momento de ingresar mis credenciales y acceder solo me muestra el aviso de suspensión y no puedo hacer nada alguien podría ayudarme con eso muchas gracias.
<img width="521" alt="capture" src="https://github.com/community/community/assets/122838288/d8712580-d0f9-4f78-b9a9-49ac17e25a01">

**Comment by samus-aran** [comment]
Hi @m-felipe1 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
